ui/page/dexclient/markets.go

```diff
@@ -216,8 +355,60 @@ func (pg *Page) readNotifications() {
 			if n.Type() == core.NoteTypeFeePayment {
 				pg.RefreshWindow()
 			}
+
+			if n.Severity() > db.Success {
+				pg.Toast.NotifyError(n.Details())
+			}
+
+		case <-pg.ctx.Done():
+			return
+		}
+	}
+}
+
+func (pg *Page) listenerMessages(host string) {
+	msgs := pg.Load.MessageSource(host)
```

Comment:
You don't need to open a websocket connection to the DEX to listen for messages, that's handled internally by Core. To know when there is an update to the order book, note that the `pg.Dexc().Core().SyncBook` method returns a `core.BookFeed` object that has a `Next()` method for receiving updates.  Not completely familiar with it but it should work for your use case. The basic idea is:

```
bookFeed, err := pg.Dexc().Core().SyncBook(nd.Host, mkt.BaseID, mkt.QuoteID)
for {
      <- bookFeed.Next()
      // Update received, reload the book
      pg.getOrderBook(nd.Host, mkt.BaseID, mkt.QuoteID)
}
```